themes/_core/js/checkout-payment.js

```diff
@@ -109,12 +111,23 @@ class Payment {
     return `.js-payment-${moduleName}`;
   }
 
+  showNativeFormErrors () {
+    $(this.termsCheckboxSelector).get(0).reportValidity();
+    $('input[name=payment-option]').get(0).reportValidity();
```

Comment:
```suggestion
$(`input[name=payment-option], ${this.termsCheckboxSelector}`).each(function() {
  this.reportValidity();
});
```
Prefer this notation please, modules can add inputs

Reply:
Agree to this notation and will do the change.

About the inputs added by modules: 
The terms checkbox is one single element selected by it's unique name. So later added checkobxes wouldn't be selected anyway.

Additionally added radio buttons for payment methods should belong to the same radio button group, shouldn't they? So calling "reportValidity()" on one group member should apply to the whole group how I understand it.

Reply:
You can chain radio buttons, and it breaks at the first one falling on error when using reportValidity.
Since it's a radio, it only occurs when the radio have the required attribute.